core/network/helpers/compressor/zstd_stream_compressor.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "compressor.h"
+namespace kagome::network {
+struct ZstdStreamCompressor : public ICompressor {
```

Comment:
`ICompressor` doesn't work with streams
```suggestion
struct ZstdCompressor : public ICompressor {
```